content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -608,8 +608,9 @@ Example:
 
 ### kubeadm's skew against the kubelet
 
-Similarly to the Kubernetes version, kubeadm can be used with a kubelet version that is the same
-version as kubeadm or one version older.
+Similarly to the Kubernetes version, kubeadm can be used with a kubelet version that is
+the same version as kubeadm or three minor versions older. (kubeadm < 1.29 may only be
+up to one minor versions newer than kubelet).
```

Comment:
the text in `( ... )` is not needed because the website docs are versioned.
leaving this to SIG docs to decided if OK.

the previous added sentence LGTM.
/lgtm